packages/expo-av/src/Audio/Sound.ts

```diff
@@ -12,10 +12,11 @@ import {
 } from '../AV';
 import ExponentAV from '../ExponentAV';
 
+type AudioInstance = number | HTMLMediaElement;
 export class Sound implements Playback {
   _loaded: boolean = false;
   _loading: boolean = false;
-  _key: number = -1;
+  _key?: AudioInstance;
```

Comment:
Like the other properties below:

```suggestion
  _key: AudioInstance | null = null;
```